web/src/components/vis/AnovaTable.vue

```diff
@@ -0,0 +1,64 @@
+<script>
+import { format } from 'd3-format';
+
+export default {
+
+  props: {
+    data: {
+      validator: prop => !prop || ('data' in prop && 'pairs' in prop),
```

Comment:
My limited personal experience suggests that this should be split into two props that can be type-validated and named in the code more easily than a composition of that same data into an object.

Reply:
(e.g., the `items` and `pairs` computed properties below could be eliminated altogether.)

Reply:
`items` and `pairs` come from `plot.data.data` and `plot.data.pairs` respectively.  plot.data can be null when the plot is invalid, and so will throw an error about accessing properties on null.  I don't think it should be `Transform.vue`'s problem to deal with this because it would push all these computed props into Transform.vue, but I would be willing to reconsider that position.

Reply:
If there's a reason to have it this way, I'm ok with it. To do it my way would require yet another `valid` prop to be computed by Transform (as well as conditionally sending in `null` for the putative `data` and `pairs` props), and maybe that's a bridge too far.

I just like the ability to simplify the local logic by splitting structured props apart, but here it looks like it would just shift the complexity to Transform.